### DISCUSSION TITLE
[Feature] Consider full path when searching for filename

### DISCUSSION THREAD

**Discussion by jeanpaulrh** [immich-app/immich#9106] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have a bunch of old digital photos that I imported into Immich through external libraries.
My photos are stored on the disk with meaningful directory names, so it would be useful to (optionally?) include full path when searching images using filenames. What do think about it?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#6519

**Reply by jeanpaulrh** [reply]
Thanks. Sorry but I didn't find it (maybe I searched through issues though). Closing it.
